README.md

```diff
@@ -1,38 +1,64 @@
 - `.slice` files, which use the [latest Slice](https://docs.icerpc.dev/slice2) syntax
 
-Additionally, this extension provides a Slice Language Server, for `.slice` files only.
+### Error Detection and Reporting
+
+Error checking is triggered every time a Slice file is saved or opened,
+and is only available for `.slice` files.
 
 ## Configuration
 
-The Slice Language Server can be configured using the following settings:
+The Slice language server that ships with this extension can be configured with the following settings:
+
+- `slice.languageServer.enabled`: A boolean indicating whether the server should be enabled. Defaults to `true`.
+
+- `slice.configurations`: An array of configuration sets (independent groups of Slice files). This allows multiple Slice projects to exist within a single repository. Each configuration set supports the following settings:
 
-- `slice.languageServer.enabled`: A boolean indicating whether the Slice Language Server should be enabled. Defaults
-  to `true`.
-- `slice.configurations`: An array of configuration sets. Configuration sets are independently compiled, allowing each
-  to define its own files and options for compilation.
-  - **Each configuration set is an object with the following properties:**
-  - `paths`: An array containing the paths to the target Slice reference files. Also supports directory paths.
-  - `addWellKnownTypes`: A boolean indicating whether to include the
-    [IceRpc well-known Slice files](https://github.com/icerpc/icerpc-slice/tree/main/WellKnownTypes) during compilation.
+    - `paths`: An array of paths to specify which Slice files should be included in this set.
+    This field is required.
+
+    - `addWellKnownTypes`: A boolean indicating whether to include the Slice definitions contained in the [IceRPC Slice](https://github.com/icerpc/icerpc-slice) repository.
+    These types are commonly used in applications utilizing Slice.
+    Defaults to `true`.
+
+If you do not specify any configuration sets, the extension will default to using the project's root directory for `paths`.
+
+**Note:** the language server only works with `.slice` files, and will ignore any `.ice` files in your project. The above settings are only meaningful for projects using `.slice` files.
 
 ### Example
 
+Below is an example `settings.json` file which configures 2 separate Slice projects:
+
 ```json
 {
+    "slice.languageServer.enabled": true,
     "slice.configurations": [
         {
             "paths": [
-                "path/to/slice/directory",
-                "path/to/other/slice/file.slice",
-                "/absolute/path/to/directory"  
-            ],
-            "addWellKnownTypes": true
+                "path/to/slice/directory"
+            ]
+        },
+        {
+            "addWellKnownTypes": false,
+            "paths": [
+                "path/to/specific/file.slice",
+                "/absolute/path/to/other/slice/files",
```

Comment:
I prefer directory over files for a sample directory name.

Reply:
Fixed.